pubspec.yaml

```diff
@@ -16,39 +16,5 @@ dependencies:
 dev_dependencies:
   flutter_test:
     sdk: flutter
-# For information on the generic Dart part of this file, see the
```

Comment:
author can be removed. This is deprecated
homepage should be the homepage of this repo so you should add the github url here